src/services/GroupService.js

```diff
@@ -68,3 +68,27 @@ export async function associateGroupAndUser(userLogin, groupId) {
 export async function dissociateGroupAndUser(userLogin, groupId) {
   return api.delete(`/groups/${groupId}/users/${userLogin}`);
 }
+
+/**
+ * Associate group and role.
+ * @param {string} groupId - Group id.
+ * @param {string} roleId - Role id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+export async function associateGroupAndRole(groupId, roleId) {
```

Comment:
move into RoleService

Reply:
and rename to associateGroup(roleId, groupId)

Reply:
done